client/src/Components/VideoCard.js

```diff
@@ -0,0 +1,50 @@
+import { useState } from "react";
+
+const VideoCard = ({ video, removeVideo }) => {
+  const [rating, setRating] = useState(video.rating);
+
+  const addVote = () => {
+    setRating(rating + 1);
+    console.log(rating);
+  };
+  const deleteVote = () => {
+    setRating(rating - 1);
+    console.log(rating);
+  };
+
+  let url = video.url.substring(video.url.indexOf("=") + 1);
```

Comment:
Ah nice way of parsing the YouTube video ID from the URL!🔥 
The only thing it might not work for all YouTube `urls`.